ig/input/fsh/PopulationRecommendation.fsh

```diff
@@ -12,13 +13,45 @@ Description: ""
 * extension contains
   ageStart 0..1 MS and
   ageEnd 0..1 MS
-* extension[ageStart].value[x] only Quantity
-* extension[ageStart].value[x] obeys age-1
-* extension[ageStart].value[x].code from http://hl7.org/fhir/ValueSet/all-time-units
+* extension contains VPPopulationRecommendationLocationExtension named location 0..* MS
+
+* extension[ageStart]
```

Comment:
ageStart and ageEnd should be the same as in VaccinationScheme.

Reply:
Addressed at da97313